deploy/files/config-production.env

```diff
@@ -1,34 +1,36 @@
-61393864626134373033653165316338626432663837623463626436303135366364336161633737
-65333332623561626636613835666461316138383837626565636232386231663965
+38393962323337636365313935656532643261333665326430333933383166333766343936663936
+3264303666613865353238316561666461313733323565340a303036366635626533663131396161
+65663763303835623562643931366231323038376538636338616161613665313934613935333065
+3934353936303739360a353162303436363534353461313733653431383338326237346162326337
+39376364666562636239363331316136326130333336393037303164396534386332306333346362
+36646637653734666336626334646665353763383564313461373830663834366261306631636462
+31333561346532366239366363653265653932303832326231653938393465626339613137353064
+38356237326265643634363331636465613839633538373032386166613933616239323135666439
+34343533346265383062353339303065393164663463356362326237346336663064666637646639
+38343065636266383238396464353439613766383537393964646237613331633531383463326663
+31653363633164316232316432646331306435333263393666303633646133336534633766353161
+65663961623966383230653333353936613065313331626639313431313063346265393938666264
+31646537363965373932633563316531393939353265663236393038303063303364613831343963
+62376365646161373130343066663931353630373561323162376334303332643564666539363761
+64613537306563363538326334653337616339613937373132663038646365626638376331643531
+37353539303462333937303665636138393463333562333836616163386539333737393264633861
+35346535303430396266663133326162383732383237633136396361663932303335646432633638
+39643138386137646431383234636339353530353637356136646234393166386633343938633361
+32383763643339366665616539643835333164376561303737313166373431613434343339616130
+37333836373163616262366562363736616164616238323339353737663064353066383130376334
+34643265376430636438363134313034346434376332303765326162316564323966373064353061
+62373832656339366437363866383834383035623131653762343530626332356633633335656433
+31623837366639663831323930356663373832343636383832653239393833643464666435633436
+30323237303662373530326334316365663361323238346439366362373365323939663739366134
+37383735636534373936643437313533363139633064373536323137643936373332366538353532
+34643231616137363834363233613037313033366161356231386333326539643033326535633061
+62616238633330636463373365636635326132326530376539313465386131353536343266306134
+31313931356634633164356562323662326363373736313765353732333736313531346539666431
+66633739643363343136393439386464376233353736303665623432313537343862666430326564
+38313666643333343931303131363462613566356235306239353064313338353839316233663131
+30636630333534386333623861373338346639393030336661663561383365613564346136316435
+32643234633339636334336664366363363462343131323735353434636435386262616236386164
+30663039313031343361653436356436353632633062653137346165333463343865653131633232
+36626634363137303262306664643734626238376333336164313761303163613165326461343535
+393264373461663830376437386138396264
```

Comment:
so all you added here is `IMPORT_CONFIG=/home/aria-bot/config.env` ?

Reply:
Yes!